src/DependencyInjection/Repository/Factory/DoctrinePhpcrOdmFactory.php

```diff
@@ -34,7 +34,7 @@ public function create(array $options)
      */
     public function getName()
     {
-        return 'doctrine_phpcr_odm';
+        return 'doctrine/phpcr-odm';
```

Comment:
again, that is the type name and won't become the repository name, which goes into the url when using resources-rest?